src/ducks/apps/index.js

```diff
@@ -672,24 +672,33 @@ export function uninstallApp(app) {
 
 async function _handleTerms(terms) {
   const { id, ...termsAttributes } = terms
-  const docId = slugify(`${id}:${termsAttributes.version}`)
+  // We use : as separator for <id>:<version>
+  const docId = `${slugify(id, { remove: /[*+~.()'"!:@]/g })}:${slugify(
+    termsAttributes.version,
+    { remove: /[*+~.()'"!:@]/g }
+  )}`
```

Comment:
It's a bit unclear what is the goal here, can we hide this behind a function like `makeTermsKey` or something ?